pkg/synthetictests/allowedalerts/all.go

```diff
@@ -62,6 +62,9 @@ func AllAlertTests(ctx context.Context, clientConfig *rest.Config, duration time
 	ret = append(ret, newAlert("machine config operator", "MCDDrainError").pending().neverFail().toTests()...)
 	ret = append(ret, newAlert("machine config operator", "MCDDrainError").firing().toTests()...)
 
+	ret = append(ret, newAlert("machine config operator", "MCDPivotError").pending().neverFail().toTests()...)
+	ret = append(ret, newAlert("machine config operator", "MCDPivotError").firing().toTests()...)
```

Comment:
Looking at your presubmits I suspect this will take out payloads, can we do neverFail for the firing as well, with a TODO comment. We can track the flake percentage to know if we're getting better.

Reply:
I see one job with this failing. I want to rerun this job to see how often this fails.

Reply:
I thought I actually saw two but I can't find another now, possibly I clicked the same link twice. Rerun sounds good lets hope it's ok.

Reply:
In the new run, this test did not fail again.

Reply:
Ok lets bring it in and we can just keep an eye on payloads. 

@deepsm007 heads up on this, if we see payloads rejected on a new MCDPivotError alert test, this is the PR.

It's possible with it being a new test, aggregation will not blow up on it?